### PR TITLE
Update troubleshooting.adoc

### PR DESCRIPTION
**Pull request by Vanadiae**
Adding a user to a group needs to unlog so change accordingly.

### CODE REVIEW

File: modules/ROOT/pages/troubleshooting.adoc [hunk 1]
```diff
@@ -99,6 +99,8 @@ to the `libvirt` group:
```
Comment (returntrip) [inline modules/ROOT/pages/troubleshooting.adoc:99]: Suggest to add the following here instead: NOTE: You will need to log off and log back in to apply these changes.

File: modules/ROOT/pages/troubleshooting.adoc [hunk 2]
```diff
@@ -99,6 +99,8 @@ to the `libvirt` group:
 
  # grep -E '^libvirt:' /usr/lib/group >> /etc/group
  # usermod -aG libvirt username
+ 
+ Now you need to unlog to apply the changes and then you can login back.
```
Comment (returntrip) [inline modules/ROOT/pages/troubleshooting.adoc:103]: See suggestion at line 100.

**Comment by Vanadiae** [comment]
I just pushed the changes.

**Review by travier — APPROVED** [review]
LGTM

**Comment by travier** [comment]
Ping @tpopela

**Comment by tpopela** [comment]
@Vanadiae thank you for the contribution and thank you @travier for the ping!

**Pull request merged**